selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/ActivityStream.java

```diff
@@ -450,15 +451,8 @@ public void viewComment(String activity, String comment, String value) {
    */
   public void addText(String text) {
     info("----Add text into activity text box-----");
-    WebElement inputText = evt.waitForAndGetElement(ELEMENT_COMPOSER_INPUT_FILED, 100000);
-    // type(ELEMENT_COMPOSER_INPUT_FILED, text, false);
-    WebElement shareButton = evt.waitForAndGetElement(ELEMENT_COMPOSER_SHARE_BUTTON);
-    WebElement workingLabel = evt.waitForAndGetElement(ELEMENT_ACTIVITY_WHAT_ARE_YOU_WORKING_LABEL);
-    ((JavascriptExecutor) testBase.getExoWebDriver().getWebDriver()).executeScript("arguments[0].textContent = '';", workingLabel);
-    ((JavascriptExecutor) testBase.getExoWebDriver().getWebDriver()).executeScript("arguments[0].textContent = '" + text + "';", inputText);
-    ((JavascriptExecutor) testBase.getExoWebDriver().getWebDriver()).executeScript("arguments[0].disabled = false;", shareButton);
-    ((JavascriptExecutor) testBase.getExoWebDriver().getWebDriver()).executeScript("arguments[0].className = 'pull-right btn btn-primary';",
-                                                                      shareButton);
+    $(byId("cke_1_contents")).click();
```

Comment:
Please create a new variable in SocialLocator instead of using string, to be compliant with other tests.